assets/js/util/sanitize.test.js

```diff
@@ -29,3 +29,19 @@ describe( 'sanitizeHTML', () => {
 		} );
 	} );
 } );
+
+describe( 'unTrailingSlashIt', () => {
+	it( 'should not change a URL without trailing slash', () => {
+		const url = 'http://example.org';
+		expect( unTrailingSlashIt( url ) ).toEqual( 'http://example.org' );
+	} );
+
+	it( 'should change a URL with trailing slash', () => {
+		const url = 'http://example.org/';
+		expect( unTrailingSlashIt( url ) ).toEqual( 'http://example.org' );
+	} );
+
+	it( 'should return null if the parameter is not a string', () => {
+		expect( unTrailingSlashIt( 1 ) ).toBeNull();
+	} );
```

Comment:
Let's add an extra test for extra trailing slashes, just to be complete:

```suggestion
	it( 'should return null if the parameter is not a string', () => {
		expect( unTrailingSlashIt( 1 ) ).toBeNull();
	} );
	
	it( 'should remove multiple trailing slashes, like the WordPress PHP function', () => {
		expect( unTrailingSlashIt( 'http://example.org/foo// ) ).toEqual( 'http://example.org/foo' );
	} );
```